public/app/core/components/AppChrome/TopSearchBar.tsx

```diff
@@ -33,15 +34,16 @@ export function TopSearchBar() {
         <TopSearchBarInput />
       </TopSearchBarSection>
       <TopSearchBarSection align="right">
+        <QuickAdd />
         {helpNode && (
-          <Dropdown overlay={() => <TopNavBarMenu node={helpNode} />}>
+          <Dropdown overlay={() => <TopNavBarMenu node={helpNode} />} placement="bottom-end">
```

Comment:
Any difference between the `overlay` prop being a function here and a React component at line 46?

Reply:
good question - i actually don't know. the examples for `Dropdown` mostly use a react component so i'll switch it to be consistent, but we explicitly allow the type for either. do you know why @torkelo ? 🤔

Reply:
@Clarity-89 @ashharrison90 no!! having be a function is essential here so these menus are rendered only when the dropdown is opened

Reply:
is should be a function on line 45 as well

Reply:
ah ok, that makes sense. then i guess the opposite question - is there any reason to ever allow a react component, or should we just make the type more restrictive and only allow functions returning react components? 🤔